app/models/reservation.rb

```diff
@@ -18,7 +18,7 @@ def overlapped_reservations
     equipment.reservations
              .where.not(id: id)
              .where('start_time < ? AND ? < end_time', end_time, start_time)
-             .where.not('status = ?', Reservation.statuses[:cancelled])
+             .where.not('status = ? OR status = ?', Reservation.statuses[:cancelled], Reservation.statuses[:rejected])
```

Comment:
Before the query, set `statuses = Reservation.statuses` then simply do 
```suggestion
             .where.not('status = ? OR status = ?', statuses[:cancelled], statuses[:rejected])
```

Reply:
Done